src/configuring/Limits_semgrep.ml

```diff
@@ -35,8 +35,11 @@ let svalue_prop_MAX_VISIT_SYM_IN_CYCLE_CHECK = 1000
  * Note that 'Time_limit.set_timeout' cannot be nested. *)
 let taint_FIXPOINT_TIMEOUT = 0.1
 
-(** Bounds the number of l-values we can track. *)
-let taint_MAX_TAINTED_LVALS = 100
+(** Bounds the number of variables we can track. *)
+let taint_MAX_TAINTED_VARS = 50
+
+(** Bounds the number of fields we can track per l-value. *)
```

Comment:
should be "per vars" in the comment, now that we talk about var instead of lval

Reply:
It's actually still "per l-value", this limits the fields in an `Obj` shape and that corresponds to an l-value. Each sub-`ref` in a var's `ref` is an l-value. Tried to clarify the comment.